source/reference/image-types.rst

```diff
@@ -77,4 +77,4 @@ Table 2 lists the currently available images that are platform specific.
      - Virtual Machine Disk for VMware\* platforms including Player,
        Workstation, and ESXi.
 
-.. _images: https://clearlinux.org/downloads
+.. _images: https://clearlinux.github.io/downloads
```

Comment:
I don't think this is supposed to be changed since it is going to the content part of the site. Anything under downloads probably needs to not change.